yletunnus/backends.py

```diff
@@ -10,8 +16,14 @@ class YleTunnusOAuth2(BaseOAuth2):
     REDIRECT_STATE = False
     ACCESS_TOKEN_METHOD = 'POST'
     DEFAULT_SCOPE = ['sub', 'email']
-    AUTHORIZATION_URL = 'https://auth.api.yle.fi/v1/authorize'
-    ACCESS_TOKEN_URL = 'https://auth.api.yle.fi/v1/token'
+
+    def _get_url(self, version='v1', *url_path):
+        environment = self.setting('YLETUNNUS_ENVIRONMENT')
+        if environment not in BASE_URLS.keys():
+            raise ImproperlyConfigured(
+                'Unknown YLE Tunnus environment. '
+                'Please set the settings variable SOCIAL_AUTH_YLETUNNUS_ENVIRONMENT to an allowed value.')
+        return '/'.join((BASE_URLS[environment], version) + url_path)
```

Comment:
Is there a good reason this just joins all the positional arguments to the url? In my opinion it would be clearer to drop the asterisk and check if the value of `url_path` is an iterable or not.

Reply:
Fixed in cd2b5f1a47945e6cf1a7b406532fbd4872a35c86 -- and at the same time it fixes a bug (the tests were incorrect too).